stories/EventCard.stories.js

```diff
@@ -0,0 +1,116 @@
+import VsGrid from '@/components/grid/Container.vue';
+import VsRow from '@/components/grid/Row.vue';
+import VsCol from '@/components/grid/Col.vue';
+import VsEventCard from '@/components/event-card/EventCard.vue';
+import VsList from '@/components/list/List.vue';
+
+export default {
+    component: VsEventCard,
+    title: 'EventCard',
+    decorators: [() => ({
+        template: '<div style="max-width:900px;"><story /></div>',
+    })],
+};
+
+const Template = (args) => ({
+    components: {
+        VsEventCard,
+        VsList,
+        VsCol,
+        VsRow,
+        VsGrid,
+    },
+    setup() {
+        return {
+            args,
+        };
+    },
+    template: `
+        <VsEventCard v-bind="args">
+                <template v-slot:event-card-header> {{args.title}} </template>
+
+                <template v-slot:event-card-date> {{ args.dates }} </template>
+
+                <template v-slot:event-card-content>
+
+                    <p class="mb-050"> {{ args.description }} </p>
+
+                    <VsRow class="mb-050">
+                        <VsCol
+                            sm="12"
+                            md="4"
+                        >
+                            <VsList
+                                unstyled
+                            >
+                                <li>
+                                    <label for="time">Time:</label>
```

Comment:
Labels should be used with form elements and not for styling text. I think what you actually want here is a description list https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl 

Since its in a slot, you could use the bootstrap utilities to style it.